exec/query.R

```diff
@@ -10,7 +10,7 @@ require(XML, quietly = TRUE)
 
 doc <- "
 USAGE:
-  query.R query <tools.yml> <project_conf.yml> <srcml_filepath> [--namespace | --file-docs | --class-docs | --variables | --packages | --functions | --imports]
+  query.R query <tools.yml> <project_conf.yml> <srcml_filepath> [--namespace | --file-docs | --class-docs | --variables | --packages | --functions | --imports | --class-names]
 
 DESCRIPTION:
   Runs a specified query on the annotated XML file and saves the output to the appropriate path.
```

Comment:
I would be more verbose here and focus more on input and output and third party tool dependencies. 

Something like "Uses srcML to parse a src code folder, as specified on the project configuration file, to parse selected content as (file,class,variable) tables. "